src/ipc/handler.c

```diff
@@ -668,6 +668,8 @@ static int ipc_pm_gate(uint32_t header)
 
 	IPC_COPY_CMD(pm_gate, ipc_get()->comp_data);
 
+	tr_info(&ipc_tr, "ipc: pm gate flags 0x%X", pm_gate.flags);
+
```

Comment:
What's the difference between %x and %X (lower case vs upper case)? I've mostly seen usages with lower case for x.

Reply:
0x%x -> 0x12ab vs 0x%X 0x12AB

Reply:
@ktrzcinx thanks I didn't know about that :).

Reply:
can we keep consistency and use the same HEX case throughout the FW. I think it's all lower case today.